luna/featurevis/featurevis.py

```diff
@@ -135,7 +66,7 @@ def visualize_filter(image, model, layer, filter_index, iterations,
     return loss, image
 
 
-def compute_loss(input_image, model, filter_index, channels_first=False):
+def compute_loss(input_image, model, filter_index):
```

Comment:
Should be removed from the docstring as well.

Reply:
Still open.

Reply:
@fuzhanrahmanian this is not fixed yet. In the function documentation, you are still referring to this variable.